python/perspective/bench/runtime/perspective_benchmark.py

```diff
@@ -184,30 +184,38 @@ def benchmark_to_format_zero(self):
         for name in (
             "arrow",
             "csv",
+            "columns",
+            "records",
         ):
-            test_meta = make_meta("to_format", "to_{}".format(name))
-            func = Benchmark(lambda: getattr(self._view, "to_{0}".format(name))(), meta=test_meta)
+            method = "to_{0}".format(name)
+            test_meta = make_meta("to_format", method)
+            func = Benchmark(getattr(self._view, method), meta=test_meta)
```

Comment:
If benchmarking results in static assets, maybe we can create an blocks example that pulls in the data.